docs/concepts/manifest-builds.md

```diff
@@ -134,6 +137,32 @@ What this means in practice is:
 - Man pages should be placed in `$out/share/man`.
 - Libraries should be placed in `$out/lib`
 
+### Sandboxed builds
+
+The script that you specify in `command` is run on your host machine by default.
```

Comment:
There are no cases that I'm aware of where the build _isn't_ run on your host machine, or am I wrong?

Reply:
It depends on terminology 🤔  I think of `sandbox=pure` builds as being analogous to running something in a virtual machine or container, where it's executing on the host _hardware_ (unlike a remote builder) but doesn't have access to the host _system_ (unlike a `sandbox=off` build). Does that match your understanding? If so, can you think of a better way to describe it?